salt/modules/pip.py

```diff
@@ -101,6 +102,7 @@
 }
 
 VALID_PROTOS = ['http', 'https', 'ftp', 'file']
+PIP_VERSION = None
```

Comment:
This seems like it could blow up in our faces if someone upgrades pip manually with `pip install -U pip`.

This I think would be a good use case for `__context__['PIP_VERSION']` it will have to be run each time the module is loaded, but it would also be able to be cleared out by a `saltutil.refresh_modules`